core/src/consensus.rs

```diff
@@ -550,8 +550,9 @@ impl Tower {
                 "Error while recording vote {} {} in local tower {:?}",
                 vote_slot, vote_hash, result
             );
+        } else {
```

Comment:
alternatively we could upgrade the above `error!` to `panic!`, if we hit it on a non hot spare validator we're likely screwed.

this will probably let us survive for some time longer though.